documentation/intro.md

```diff
@@ -2,7 +2,7 @@
 ## What is Catlas?
 Catlas is an automated framework for mass inference of adsorption energies.
 
-It relies on the [infrastructure](https://github.com/Open-Catalyst-Project/Open-Catalyst-Dataset) that was built out for the OC20 dataset\cite{OC20} to enumerate adsorbate-surface configurations. The workflow starts from relaxed bulk structures. The bulk structures which were included in OC20 are included in the repo in the `catlas/bulk_structures` folder. This set of bulks was originally taken from the Materials Project Database\cite{MP} and then relaxed with RPBE. Optionally you can add your own bulk file. For more info, see advanced use. Slabs are cut from the bulk structures using pymatgen\cite{pymatgen}. From there, adsorbates are placed on the slabs using CatKit\cite{catkit}. Adsorbates from OC20 are included here in the `catlas/adsorbate_structures` folder. Optionally you can add your own adsorbate file. For more info, see advanced use. Finally, ML inference is performed using the [infrastructure](https://github.com/Open-Catalyst-Project/ocp) for OCP models. To expedite this process, catlas parallelizes enumeration and inference over many CPUs and GPUs using dask\cite{dask}. Pretrained model checkpoint files may be downloaded [here](https://github.com/Open-Catalyst-Project/ocp/blob/main/MODELS.md), but have been ommited from this repo because of their file size.
+It relies on the [infrastructure](https://github.com/Open-Catalyst-Project/Open-Catalyst-Dataset) that was built out for the OC20 dataset[@OC20] to enumerate adsorbate-surface configurations. The workflow starts from relaxed bulk structures. The bulk structures which were included in OC20 are included in the repo in the `catlas/bulk_structures` folder. This set of bulks was originally taken from the Materials Project Database[@MP] and then relaxed with RPBE. Optionally you can add your own bulk file. For more info, see advanced use. Slabs are cut from the bulk structures using pymatgen[@pymatgen]. From there, adsorbates are placed on the slabs using CatKit[@catkit]. Adsorbates from OC20 are included here in the `catlas/adsorbate_structures` folder. Optionally you can add your own adsorbate file. For more info, see advanced use. Finally, ML inference is performed using the [infrastructure](https://github.com/Open-Catalyst-Project/ocp) for OCP models. To expedite this process, catlas parallelizes enumeration and inference over many CPUs and GPUs using dask[@dask]. Pretrained model checkpoint files may be downloaded [here](https://github.com/Open-Catalyst-Project/ocp/blob/main/MODELS.md), but have been ommited from this repo because of their file size.
```

Comment:
Do other people have access to our bulk_structures folder and models? Would they have to download and link them manually or will they be downloaded with catlas?

Reply:
I think this is covered in install, the bulk file is in the repo, the models they are instructed to download and told where